tools/hive-bigquery/properties_reader.py

```diff
@@ -0,0 +1,33 @@
+"""Properties Reader module"""
```

Comment:
Apache header. 
Also Why are you making this Properties Reader and creating this dependency on a json file on disk?